src/Pages/woosukkim/Login/Login.js

```diff
@@ -1,11 +1,77 @@
-import React from "react";
-import "./Login.scss";
-import { withRouter } from "react-router-dom";
-
-class Login extends React.Component {
-  render() {
-    return <div></div>;
-  }
+import React, { Component } from "react";
+import "../Login/Login.scss";
+import { Link } from "react-router-dom";
+
+class Login extends Component {
+	state = {
+		accountValue: "",
+		passwordValue: "",
+	};
+
+	handleAccValue = (e) => {
+		console.log(this.state.accountValue.length);
+		this.setState({
+			accountValue: e.target.value,
+		});
```

Comment:
비구조화를 알려드리겠습니다! 좀 더 간결하게 보이기 위해 사용합니다!
```suggestion
                const {value} = e.target;
		this.setState({
			accountValue: value,
		});
```

Reply:
비구조화 사용하였습니다